cmd/zoekt-sourcegraph-indexserver/cleanup.go

```diff
@@ -59,6 +60,16 @@ func cleanup(indexDir string, repos []uint32, now time.Time, shardMerging bool)
 		delete(trash, repo)
 	}
 
+	for repo, _ := range tombtones {
```

Comment:
the other sections have a little comment explaining the why we do this, why at this point/etc and extra info. Can this also have one?